lib/cursor.js

```diff
@@ -768,6 +769,29 @@ class Cursor extends CoreCursor {
     }
   }
 
+  /**
+   * Set the ReadPreference for the cursor.
+   * @method
+   * @param {(string|ReadConcern)} readConcern The new read preference for the cursor.
+   * @throws {MongoError}
+   * @return {Cursor}
+   */
+  setReadConcern(readConcern) {
```

Comment:
nit: I think that `withReadConcern` reads better, thoughts?

Reply:
I think it should be consistent with `setReadPreference`.

Reply:
I'm thinking we should change `setReadPreference` to `withReadPreference` :smile:

Reply:
This is 3.6 that would be a breaking change here, are you ok with that?

Reply:
I was thinking in 3.6 we would provide both, and deprecate `setReadPreference`, advising users to use `withReadPreference` instead